make image stuck time to time when download start.sh and restart.sh

/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]

build stuck here time to time though I can access internet..
I searched and didn't found start.sh and restart.sh usage .. so can we remove them? 
```
---> cb7ce8bbd8c7
Step 8/18 : COPY ./ ./
 ---> 46cd4b39728b
Step 9/18 : RUN wget --output-document /restart.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/restart.sh  &&     wget --output-document /start.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/start.sh &&     chmod +x /start.sh && chmod +x /restart.sh
 ---> Running in f65270b870e9
^CMakefile:205: recipe for target 'docker-build' failed
make: *** [docker-build] Interrupt

```

**What did you expect to happen:**


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Cluster API Provider OpenStack version (Or `git rev-parse HEAD` if manually built):
- Cluster-API version:
- OpenStack version:
- Minikube/KIND version:
- Kubernetes version (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

--- a/Dockerfile
+++ b/Dockerfile
@@ -30,10 +30,6 @@ RUN go mod download
 # Copy the sources
 COPY ./ ./
 
-RUN wget --output-document /restart.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/restart.sh  && \
-    wget --output-document /start.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/start.sh && \
-    chmod +x /start.sh && chmod +x /restart.sh
-
 # Build
 ARG ARCH
 ARG LDFLAGS

